server/app.js

```diff
@@ -13,22 +13,19 @@ const authRouter = require("./routes/auth");
 const userRouter = require("./routes/user");
 const notificationRouter = require('./routes/notifications');
 const imagesRouter = require("./routes/imageUpload");
+const { initSocketServer } = require("./utils/socketsServer/socketServer");
 
 const { json, urlencoded } = express;
 
 connectDB();
 const app = express();
 const server = http.createServer(app);
 
-const io = socketio(server, {
-  cors: {
-    origin: "*"
-  }
-});
-
-io.on("connection", socket => {
-  console.log("connected");
-});
+// const io = socketio(server, {
+//   cors: {
+//     origin: "*"
+//   }
+// });
```

Comment:
If this code is no longer needed you should delete it rather than comment it out as commented code can quickly clutter an application

Reply:
Of course, you're right. Fixed. I'll get rid of it.